### PR TITLE
Update Total Nutrients class

### PR DESCRIPTION
**Pull request by VanPaitin**
Update the total nutrients class

Currently, we can only access the calorie of a count of a nutrient. We need to be able to access values for sodium, fat, and sugar.

This change addresses the need by:

* Adding helper methods to return sugar, fat and sodium, nutrient value objects
* Adding helper methods to return label, quantity, and unit of a nutrient value object

Resolves issue #12

### CODE REVIEW

File: lib/edamam-ruby/nutrient_value.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module Edamam
+  class NutrientValue
+    def initialize(value)
+      @value = value
```
Comment (chriswoodford) [inline lib/edamam-ruby/nutrient_value.rb]: is `@value` being used as hash?

File: lib/edamam-ruby/nutrient_value.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module Edamam
+  class NutrientValue
```
Comment (chriswoodford) [inline lib/edamam-ruby/nutrient_value.rb]: just `Nutrient` might be a better class name. `Value` isn't adding any clarity to the name. The edamam API returns `totalNutrients`, so it makes sense that the singular of that collection would be `Nutrient`

**Review by chriswoodford — CHANGES_REQUESTED** [review]

File: lib/edamam-ruby/nutrient_value.rb (hunk 1 above)
Comment (andela-mpitan) [inline lib/edamam-ruby/nutrient_value.rb]: Yeah, @value is a hash

Comment (chriswoodford) [inline lib/edamam-ruby/nutrient_value.rb]: you should never store a single instance variable in a class as a hash. use explicit properties for every value you need. you can use meta programming to reduce the burden of setting variables. or consider whether a gem like https://github.com/solnic/virtus is justified

**Review by chriswoodford — CHANGES_REQUESTED** [review]

File: spec/food_database_spec.rb [hunk 3]
```diff
@@ -38,7 +38,57 @@
         response = food_database.nutritional_data("1 large chicken")
         result = response.kilocalories
 
-        expect(result).to eq(115.96)
+        expect(result).to be_an_instance_of Edamam::Nutrient
```
Comment (chriswoodford) [inline spec/food_database_spec.rb]: `expect(result).to be_an_instance_of(Edamam::Nutrient)` 

don't forget to use the parentheses in your expectations. in general, the only time to not use the parenthesis is if it's a DSL and you do not capture any return value

Comment (chriswoodford) [inline spec/food_database_spec.rb]: same for line 29

**Review by chriswoodford — CHANGES_REQUESTED** [review]
just 2 small syntax things and this will be good to go

**Review by chriswoodford — APPROVED** [review]

**Pull request merged**